.teamcity/_Self/Templates/BaseTemplate.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed to the Ed-Fi Alliance under one or more agreements.
+ * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ * See the LICENSE and NOTICES files in the project root for more information.
+ */
+
+// This base class should be used by all templates
```

Comment:
so this brings up a consistency question. should we be using c style headers or c++ headers. we should choose one over the other. In my opinion we should the the c++ headers, and update the header template settings in vscode.

Reply:
I just used what came out of the box. Now that you've pointed it out, it seems better to apply the "csharp" setting and use `//`. I'll make that change in the configuration and in each file.